website/snippets/_sl-test-and-query-metrics.md

```diff
@@ -4,44 +4,41 @@ This section explains how you can test and run MetricFlow commands with dbt Clou
 
 Support for running [MetricFlow commands](/docs/build/metricflow-commands) in the dbt Cloud IDE is not available but is coming soon. 
 
-You can use the **Preview** or **Compile** buttons in the IDE to run semantic validations and make sure your metrics are defined. Alternatively, you can run commands with the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation) or with SQL client tools like DataGrip, DBeaver, or RazorSQL. 
+You can use the **Preview** or **Compile** buttons in the IDE to run semantic validations and make sure your metrics are defined. Alternatively, you can run commands with the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation) or with SQL client tools like DataGrip, DBeaver, or RazorSQL.
 
 :::
 
 <Tabs>
 
-
 <TabItem value="cloud" label="dbt Cloud">
 
-
 This section is for people using the dbt Cloud CLI (support for dbt Cloud IDE is coming soon). With dbt Cloud:
 
 - You can run MetricFlow commands after installing the dbt Cloud CLI. They're integrated with dbt Cloud so you can use them immediately.
 - Your account will automatically manage version control for you.
 
-To get started: 
-
-1. Make sure you've installed the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation). 
-2. Navigate to your dbt project directory.
-3. Run a dbt command, such as `dbt parse`, `dbt run`, `dbt compile`, or `dbt build`. If you don't, you'll receive an error message that begins with: "ensure that you've ran an artifacts...."
-   - MetricFlow builds a semantic graph and generates a `semantic_manifest.json` file in dbt Cloud, which is stored in the `/target` directory. If using the Jaffle shop example, run `dbt seed && dbt run` to ensure the required data is in your data platform before proceeding.
+Refer to the following steps to get started:
 
-4. Run `dbt sl --help` to confirm you have MetricFlow installed and that you can view the available commands.
-5. Run `dbt sl query --metrics <metric_name> --group-by <dimension_name>` to query the metrics and dimensions. For example, `dbt sl query --metrics order_total --group-by metric_time`
-6. Verify that the metric values are what you expect. To further understand how the metric is being generated, you can view the generated SQL if you type `--compile` in the command line.
-7. Commit and merge the code changes that contain the metric definitions.
+1. **Installation** &mdash; Make sure you've installed the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation). Then, navigate to your dbt project directory.
```

Comment:
i see what you're doing here but the construction, layout, and font decoration is uncommon for a set of steps. consider constructing this slightly differently. 

```suggestion
1. Install the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation) (if you haven't already). Then, navigate to your dbt project directory.
```

Reply:
fair point, i played around with it and knew it was diff to the others but wasn't sure if having the steps bolded would help orient users so went with it for the greater good.